src/Symfony/Contracts/Service/ServiceSubscriberTrait.php

```diff
@@ -43,7 +43,7 @@ public static function getSubscribedServices(): array
             }
 
             if (self::class === $method->getDeclaringClass()->name && ($returnType = $method->getReturnType()) && !$returnType->isBuiltin()) {
-                $services[self::class.'::'.$method->name] = '?'.($returnType instanceof \ReflectionNamedType ? $returnType->getName() : $type);
+                $services[self::class.'::'.$method->name] = '?'.($returnType instanceof \ReflectionNamedType ? $returnType->getName() : $returnType);
```

Comment:
The variable `$type` is never defined in this class. I assume we mean `$returnType`.